locations/spiders/wash_n_dry_fr.py

```
@@ -0,0 +1,52 @@
+    start_urls = ["https://www.washndry-concept.com/nos-laveries/"]
+    locations_key = "laveries"
+    drop_attributes = ["state"]
+
+    def extract_json(self, response):
+        data = chompjs.parse_js_object(response.xpath('//script[contains(text(), "var Theme = ")]/text()').get())
+        return chompjs.parse_js_object(data["laveries"])
+
+    def post_process_item(self, item, response, location):
+        item["ref"] = location["url"]
+        item["branch"] = (
+            (item.pop("name", "") or "").removeprefix("LAVERIE ").removeprefix("WASH’N DRY ").removeprefix("– ")
```

Ruff (0.16.3)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)